webapp/src/reducers/index.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+import {combineReducers} from 'redux';
+
+import ActionTypes from '../action_types';
+
+function connected(state = false, action) {
+    switch (action.type) {
+    case ActionTypes.AUTH_CONNECTED:
+      return action.data;
+    default:
+      return state;
+    }
+}
+
+function disconnected(state = true, action) {
+    switch (action.type) {
+    case ActionTypes.AUTH_CONNECTED:
+      return false;
+    case ActionTypes.AUTH_DISCONNECTED:
+      return true;
+    default:
+      return state;
+    }
+}
+
+function authenticated(state = {user : {}, session: {}}, action) {
+    switch (action.type) {
+    case ActionTypes.AUTH_CONNECTED:
+      return action.data;
+    case ActionTypes.AUTH_DISCONNECTED:
+      return true;
```

Comment:
When `AUTH_DISCONNECTED` `authenticated` is `true`?

Reply:
ha! oops, I'll update that.